docs/mdbook/src/android/getting-started-guide.md

```diff
@@ -0,0 +1,118 @@
+# Quickstart
+
+1. Add bintray Maven repositories to your project-level Gradle file (usually `<project>/<app-module>/build.gradle`).
+
+    ```gradle
+        allprojects {
+            repositories {
+                ...
+                mavenCentral()                
+            }
+        }
+    ```
+
+2. Add the library as a dependency into your module Gradle file (usually `<project>/<app-module>/build.gradle`). Please replace `<version>` with a valid MapLibre GL version (e.g.: `org.maplibre.gl:android-sdk:10.0.2`).
+
+    ```gradle
+        dependencies {
+            ...
+            implementation 'org.maplibre.gl:android-sdk:<version>'
+            ...
+        }
+    ```
+
+3. Sync your Android project with Gradle files.
+
+4. Add a `MapView` to your layout XML file (usually `<project>/<app-module>/src/main/res/layout/activity_main.xml`).
+
+    ```xml
```

Comment:
Could you remove the indentation in all code blocks so people more easily can copy paste?

Reply:
I removed the indentation in the code blocks in this commit eae556ea3b61954471a90396d88d11bb6dc18a66